src/popover.tsx

```diff
@@ -116,8 +116,11 @@ const getWidthDesktop = (customWidth?: number): number => (customWidth ? customW
 const getPosition = (position: Position = defaultPositionDesktop, isTabletOrSmaller: boolean) =>
     isTabletOrSmaller && (position === 'left' || position === 'right') ? defaultPositionMobile : position;
 
-const getWidth = (isTabletOrSmaller: boolean, width?: number): number =>
-    isTabletOrSmaller ? window.innerWidth - marginLeftRightMobile * 2 : getWidthDesktop(width);
+const getWidth = (isTabletOrSmaller: boolean, isIos: boolean, width?: number): number =>
+    // in iOS, when the webview is rendered offscreen (eg. acccount tab), window.innerWidth value wrong, it returns strange values like 0 or 80.
+    isTabletOrSmaller
+        ? (isIos ? window.screen.width : window.innerWidth) - marginLeftRightMobile * 2
+        : getWidthDesktop(width);
```

Comment:
Not very proud of this fix, but I was unable to find a better solution. 

I tried to listen to visibility changes and recalculate but doesn't work on the first render and the bug is reproduced